participants/vof.md

```diff
@@ -0,0 +1,58 @@
+# VOF(Video Over Filecoin)
+
+## (0) Project Name and Team Slack Handles
+
+Project Name:VOF (Video Over Filecoin)  
+Slack:kwuaint
+
+## (1) Brief Project Description
+
+*The Video Over Filecoin (VOF) is a free video share service website, using blockchain to provide a platform that will help to share interesting video or movie between each others.Relies on infinite and reliable blockchain storage,People can upload and share videos for free or for a fee.*
+
+## (2) Link to Project UI
+
+https://vof.ipfs9.com
+
+## (3) What does your application/UI do?
+
+*The website can receive video and metadata uploaded by people.Others can then pay or watch the uploaded videos for free.*
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+*Various*
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+*We will not collect any data, and store it in the Filecoin network. In the IPFS network, there is no complete data in our application, and the data uploaded to the IPFS network will be reviewed by our staff.*
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+
+*At least 100T of media data.*
+
+## (7) How are you structuring the data?
+
+*We plan to store the hole data on filecoin storage，but we are still try to put some real-time data in a key-value pair database.*
+
+## (8) What pre-processing are you doing before ingesting the data?
+
+*This is decided by filecoin storage system.*
+
+## (9)  What tech stack will you use to store the data?
+
+*We plan to use mysql or elanticsearch to speed up search.*
```

Comment:
On the storage side, will you be using Lotus or Powergate?

Reply:
we use the lotus to store the real video, and use the mysql to speed up the buffering and searching.